flexmeasures/data/models/planning/storage.py

```diff
@@ -27,7 +27,7 @@ class StorageScheduler(Scheduler):
     __version__ = "1"
     __author__ = "Seita"
 
-    def compute_schedule(
+    def compute(
```

Comment:
This method needs to be deprecated and sunset, too. Otherwise, other code (e.g. plugins) that calls `StorageScheduler.compute_schedule()` will call `Scheduler.compute_schedule()` and end up calling `Scheduler.compute()`, which does nothing.

Reply:
You're right!
I've added an explicit sunset of the method and the function `compute_schedule` will call `compute`.